userspace/engine/formats.cpp

```diff
@@ -100,115 +95,84 @@ int falco_formats::free_formatter(lua_State *ls)
 
 	if(source == "syscall")
 	{
-		sinsp_evt_formatter *formatter = (sinsp_evt_formatter *) lua_topointer(ls, -1);
+		sinsp_evt_formatter *formatter = (sinsp_evt_formatter *)lua_topointer(ls, -1);
 		delete(formatter);
 	}
 	else
 	{
-		json_event_formatter *formatter = (json_event_formatter *) lua_topointer(ls, -1);
+		json_event_formatter *formatter = (json_event_formatter *)lua_topointer(ls, -1);
 		delete(formatter);
 	}
 
 	return 0;
 }
 
-int falco_formats::free_formatters(lua_State *ls)
+void falco_formats::free_formatters()
```

Comment:
Could we verify this is not a leftover?

Because it was used by Lua outputs and now I don't see it used anymore. Maybe I'm wrong as usual.

Just double-check please.